post-result.py

```diff
@@ -22,6 +23,7 @@
 JENKINS_URL = sys.argv[2]
 BUILD_URL = sys.argv[3]
 BUILD_NAME = sys.argv[4]
+PUBLIC_JENKINS_URL = sys.argv[5]
```

Comment:
The 5 parameters are required.
I will add a function to parse all the arguments which will also add comments and check arguments